packages/SideNavigation/SubMenu/SubMenu.jsx

```diff
@@ -50,18 +78,18 @@ class SubMenu extends React.Component {
     subMenuLink: true,
   }
 
-  subMenuClasses = joinClassNames(
-    styles.buttonSubMenu,
-    this.props.active ? styles.active : styles.buttonDefault
-  )
-
   render() {
     const { children, label, isOpen, active, ...rest } = this.props
 
     return (
-      <div className={styles.mainDiv}>
-        <button {...safeRest(rest)} onClick={this.handleClick} className={this.subMenuClasses}>
-          <Box vertical={3} inline horizontal={2} dangerouslyAddClassName={styles.space}>
+      <div>
```

Comment:
Do you still need this `<div>` here? Can this be replaced with `React.Fragment`?

Reply:
Yes it can